source/slang/slang-lower-to-ir.cpp

```diff
@@ -7446,7 +7446,7 @@ IRInst* getOrEmitDebugSource(IRGenContext* context, PathInfo path)
         content = UnownedStringSlice((char*)outBlob->getBufferPointer(), outBlob->getBufferSize());
     IRBuilder builder(*context->irBuilder);
     builder.setInsertInto(context->irBuilder->getModule());
-    auto debugSrcInst = builder.emitDebugSource(path.foundPath.getUnownedSlice(), content);
+    auto debugSrcInst = builder.emitDebugSource(path.foundPath.getUnownedSlice(), content, false);
```

Comment:
why am I not seeing any place where we create a debug source with isIncludedFile = true?